content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,263 @@
+#. Enter vehicle :doc:`car model(s) <../../../../hr/fleet/models>`.
+#. Enable :ref:`necessary features <inventory/shipping_receiving/inventory-features>` in the
+   **Inventory** app.
+#. Set up :ref:`vehicles as delivery methods
+   <inventory/shipping_receiving/delivery-method-for-carrier>`.
+#. Create :ref:`dock locations <inventory/shipping_receiving/docks>`.
+
+.. _inventory/shipping_receiving/vehicle-capacity:
+
+Vehicle capacity
+----------------
+
+To configure the vehicle capacity, go to :menuselection:`Fleet app --> Configuration -->
+Categories`.
+
+Next, click :guilabel:`New` to add a new category, or click into an existing category to modify it.
+In the :guilabel:`Name` field, enter the type of vehicle (e.g., `Pick-up truck`, `Van`, or `Cargo
+truck`). Then, enter the vehicle capacity in the :guilabel:`Max Weight` (in kilograms) and
+:guilabel:`Volume` (in cubic meters) fields. `
+
+.. note::
+   The units of measurement for vehicle capacity are assigned at the global level in the
+   **Settings** app, in the :guilabel:`Units of Measure` section.
+
+   .. image:: dispatch/settings.png
+      :alt: Show units of measure settings.
+
+.. seealso::
+   :ref:`Vehicle category <fleet/categories>`
+
+.. image:: dispatch/categories.png
+   :alt: Vehicle categories with defined weight and volume.
+
+Car model
+---------
+
+Configuring a vehicle's car model is required when adding vehicles in Odoo. Ensure the correct
+*Category* is selected for a car model. Doing so automatically applies weight and volume capacities
+to all vehicles of that type.
+
+To configure, navigate to :menuselection:`Fleet app --> Configuration --> Models`.
+
+.. seealso::
+   :doc:`Create car model <../../../../hr/fleet/models>`
+
+From the :guilabel:`Models` list, select an existing model, or click :guilabel:`New` in the top-left
+corner to create a new model. Then, set the relevant :guilabel:`Category` field to the relevant
+vehicle category.
+
+.. _inventory/shipping_receiving/inventory-features:
+
+Inventory settings
+------------------
+
+Next, go to :menuselection:`Inventory app --> Configuration --> Settings`, and enable the required
+features for dispatch management.
+
+In the :guilabel:`Operations` section, tick the :guilabel:`Batch, Wave & Cluster Transfers`
+checkbox to prepare batches of orders for delivery.
+
+In the :guilabel:`Shipping` section, tick the :guilabel:`Delivery Methods` and :guilabel:`Dispatch
+Management System` checkboxes. Doing so allows specific vehicles to be :ref:`set as carriers
+<inventory/shipping_receiving/delivery-method-for-carrier>`.
+
+In the :guilabel:`Warehouse` section, tick the :guilabel:`Storage Locations` checkbox to assign
+specific locations in the warehouse as loading zones for delivery trucks.
+
+Then, once all the configurations are complete, be sure to click :guilabel:`Save`.
+
+.. _inventory/shipping_receiving/delivery-method-for-carrier:
+
+Delivery method
+---------------
```

Comment:
Just double checking- it isnt stated anywhere in this section whether or not to publish the newly created/configured delivery method. Is it necessary to state this?

Reply:
Thanks for being thorough :blush: Publishing the delivery method is necessary if it's available on the website. I don't think it's necessary to mention this, since it's more related to ecommerce stuff and is mentioned on the delivery method doc